Sources/SnapshotPreviewsCore/ExpandingViewController.swift

```diff
@@ -20,13 +20,19 @@ public final class ExpandingViewController: UIHostingController<EmergeModifierVi
     rootView.supportsExpansion
   }
 
+  private let HeightExpansionTimeLimitInSeconds: Double = 30
+
   private var didCall = false
   var previousHeight: CGFloat?
 
   var heightAnchor: NSLayoutConstraint?
   private var widthAnchor: NSLayoutConstraint?
 
-  public var expansionSettled: ((EmergeRenderingMode?, Float?, Bool?) -> Void)? {
+  private var startTime: Date?
+  private var timer: Timer?
+  private var elapsedTime: TimeInterval = 0
```

Comment:
Can we delete this and just make it a local var in the timeout handler? It doesn't seem to be used outside of that